packages/deployer/internal/router-helper.js

```diff
@@ -14,11 +13,7 @@ function toPrivateConstantCase(name) {
 }
 
 function getRouterSource() {
-  const routerPath = path.join(
-    relativePath(hre.config.paths.sources, hre.config.paths.root),
-    'Router.sol'
-  );
-
+  const routerPath = path.join(hre.config.paths.sources, 'Router.sol');
```

Comment:
No need of making relative the path here, and it broke when being ran with a custom folder.